### ISSUE TITLE
OS X Installation issue

### ISSUE THREAD

**Issue by andreysamode** [lpenz/atdtool#10]
Hello,

I'm attempting to install this on OS X.

When I type "pip3 search atdtool", I get:
`atdtool     - After the Deadline command-line tool`

When I type "pip3 install atdtool", I get:
`Collecting atdtool`
`Could not find a version that satisfies the requirement atdtool (from versions: )`
`No matching distribution found for atdtool`

Any ideas?

Thanks.

P.S. Below is a more detailed output with -vvv:

`pip3 -vvv install atdtool`
Collecting atdtool
  1 location(s) to search for versions of atdtool:
- https://pypi.python.org/simple/atdtool/
  Getting page https://pypi.python.org/simple/atdtool/
  Looking up "https://pypi.python.org/simple/atdtool/" in the cache
  Current age based on date: 228
  Freshness lifetime from max-age: 600
  Freshness lifetime from request max-age: 600
  The response is "fresh", returning cached response
  600 > 228
  Analyzing links from page https://pypi.python.org/simple/atdtool/
  Could not find a version that satisfies the requirement atdtool (from versions: )
  Cleaning up...
  No matching distribution found for atdtool
  Exception information:
  Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pip/basecommand.py", line 209, in main
    status = self.run(options, args)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pip/commands/install.py", line 299, in run
    requirement_set.prepare_files(finder)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pip/req/req_set.py", line 360, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pip/req/req_set.py", line 512, in _prepare_file
    finder, self.upgrade, require_hashes)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pip/req/req_install.py", line 273, in populate_link
    self.link = finder.find_requirement(self, upgrade)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pip/index.py", line 489, in find_requirement
    'No matching distribution found for %s' % req
  pip.exceptions.DistributionNotFound: No matching distribution found for atdtool

**Comment by lpenz** [comment]
No idea...
I've never been able to test under OSX.
I'll check to see if there is anything wrong with the package, and integrate a travis test for it or something like that

**Comment by lukasreinfurt** [comment]
**TL;DR:** try `pip install git+https://github.com/lpenz/atdtool` instead (git has to be installed) ([more details](https://pip.pypa.io/en/latest/reference/pip_install/#vcs-support))

For others that might have the same problem: I ran into the same issue on Ubuntu today. I did some digging and (with my limited understanding of the python ecosystem) I think it boils down to this:

The [atdtool page on pypi](https://pypi.python.org/pypi/atdtool/1.3) is not set up as a repository as running `pip search atdtool` would suggest. You can also see this if you visit <https://pypi.python.org/simple/atdtool/> as there are no links available and no binaries are linked on the main page. This seems to be a common problem:

> Historically PyPI did not have any method of hosting files nor any method of automatically retrieving installables, it was instead focused on providing a central registry of names, to prevent naming collisions, and as a means of discovery for finding projects to use. In the course of time setuptools began to scrape these human facing pages, as well as pages linked from those pages, looking for things it could automatically download and install. [...] this blurring has caused confusion for the end users of both of these roles and this has in turn caused ire between people attempting to use PyPI in different capacities, most often when end users want to use PyPI as a repository but the author wants to use PyPI solely as an index.

See [here for more details](http://legacy.python.org/dev/peps/pep-0470/).

Installing atdtool with pip directly from the git repository as described above worked for me. Maybe this could be clarified somewhere in the readme.
